hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/pipeline/StreamStageImpl.java

```diff
@@ -97,6 +100,11 @@ public StreamStageImpl(
         return attachFlatMapUsingContext(contextFactory, flatMapFn);
     }
 
+    @Nonnull @Override
+    public <R> StreamStage<R> rollingAggregation(@Nonnull AggregateOperation1<? super T, ?, ? extends R> aggrOp) {
```

Comment:
The name `rollingAggregation` is not in line with other methods that are called `aggregate` and not `aggregation`. It is generally not in line with operation names being verbs.